AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -66,8 +66,27 @@ private extension SessionHeaderView {
                     }
                 }
             }
-            Text("\(session.type?.description ?? SessionType.unknown("").description): \(session.deviceType?.description ?? "")")
-                .font(Font.moderate(size: 13, weight: .regular))
+            // As long as we get session.deviceType = nil we should handle somehow showing those devices which were used to record
+            // [|(-)   (-)|]    |-------------------|
+            //  |   ___   |  -- | You, do something |
+            //  |_________|     |-------------------|
+            // so the idea at leat for now is this below
+            #warning("Fix - do it better")
+            if session.type?.description == "Fixed" {
+                Text("\(session.type?.description ?? SessionType.unknown("").description): AirBeam3")
+                    .font(Font.moderate(size: 13, weight: .regular))
+            } else if session.type?.description == "Mobile" {
+                if session.allStreams!.count > 1 {
+                    Text("\(session.type?.description ?? SessionType.unknown("").description): AirBeam3")
+                        .font(Font.moderate(size: 13, weight: .regular))
+                } else {
+                    Text("\(session.type?.description ?? SessionType.unknown("").description): Phone Mic")
+                        .font(Font.moderate(size: 13, weight: .regular))
+                }
```

Comment:
I think this if could be simpler. Assuming that in MVP we use either AB3 or mic we could do `if session.deviceType == .MIC` and return "Phone's mic" and else return "AirBeam3" - as simple as that ;D

Reply:
Wow. Haven't thought. cool!

Reply:
I know now, it is not working sadly because we get always nil ad .deviceType